spec/electra/attestation.go

```diff
@@ -29,10 +29,12 @@ import (
 
 // Attestation is the Ethereum 2 attestation structure.
 type Attestation struct {
-	AggregationBits bitfield.Bitlist `ssz-max:"131072"`
+	// bitfield.Bitlist has size of n bits + 1 length bit, e.g. an 8 bit list will require a 2 byte array.
+	AggregationBits bitfield.Bitlist `ssz-max:"16385"`
 	Data            *phase0.AttestationData
-	Signature       phase0.BLSSignature  `ssz-size:"96"`
-	CommitteeBits   bitfield.Bitvector64 `dynssz-size:"MAX_COMMITTEES_PER_SLOT/8" ssz-size:"8"`
+	Signature       phase0.BLSSignature `ssz-size:"96"`
+	// bitfield.Bitvector64 is an 8 byte array so dynamic sizing doesn't make sense.
+	CommitteeBits bitfield.Bitvector64 `ssz-size:"8"`
```

Comment:
This is an interesting one.  `Bitvector64` is used here because the mainnet value for `MAX_COMMITTEES_PER_SLOT` is 64, but that isn't the case with the minimal spec.  I think that we go ahead with this as-is, but need to flag it with the dynssz developer to see how they want to handle this (after this has been merged).